runners/core-java/src/main/java/org/apache/beam/runners/core/SimpleDoFnRunner.java

```diff
@@ -930,7 +935,7 @@ public Timer timer(String timerId) {
       try {
         TimerSpec spec = (TimerSpec) signature.timerDeclarations().get(timerId).field().get(fn);
         return new TimerInternalsTimer(
-            window, getNamespace(), timerId, spec, stepContext.timerInternals());
+            window, getNamespace(), timerId, spec, fireTimestamp(), stepContext.timerInternals());
```

Comment:
I think this should be timestamp(), not fireTimestamp().

Reply:
@reuvenlax  done